checka11y.css

```diff
@@ -26,6 +28,7 @@ html:not( [lang] ) body::before {
 }
 
 /**
+>>>>>>> master
```

Comment:
Could we remove this line?

Reply:
Wait how did these diffs sneak in there? 😅